tests/fertiscan/db/delete_inspection/test_delete_organization_info.py

```diff
@@ -123,7 +125,8 @@ def test_delete_organization_information_with_shared_location(self):
         # Delete the first organization information
         self.cursor.execute(
             """
-            SELECT delete_organization_information(%s);
+            DELETE FROM organization_information
```

Comment:
This could be solved in another issue, but writing queries in tests makes them harder to maintain. I advise creating python function and use them, this makes the maintenance more dynamic and less redundant

Reply:
Created https://github.com/ai-cfia/ailab-datastore/issues/158 to track this.